dlt/common/libs/sqlglot.py

```diff
@@ -7,6 +7,7 @@
 try:
```

Comment:
we can move this try catch block now, sqlglot is now a main dependency of dbt and will always be availble